lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -4229,6 +4371,12 @@ class ConvertTorchToLinalg
     target.addIllegalOp<AtenTensorIntOp, AtenTensorFloatOp>();
     patterns.add<ConvertAtenArangeStartStepOp>(typeConverter, context);
     target.addIllegalOp<AtenArangeStartStepOp>();
+    patterns.add<ConvertAtenThresholdOp<AtenThresholdOp>>(typeConverter, context);
+    target.addIllegalOp<AtenThresholdOp>();
+    patterns.add<ConvertAtenThresholdOp<AtenThreshold_Op>>(typeConverter, context);
+    target.addIllegalOp<AtenThreshold_Op>();
```

Comment:
We shouldn't need to handle the `_` case because ReduceOpVariants should have converted the one with `_` to the one with value semantics.